pkg/config/config.go

```diff
@@ -11,8 +13,6 @@ import (
 const (
 	//KubePolicyNamespace default kyverno namespace
 	KubePolicyNamespace = "kyverno"
-	//WebhookServiceName default kyverno webhook service name
-	WebhookServiceName = "kyverno-svc"
```

Comment:
don't we need a default here?

Reply:
Yeah, we are setting default in the getwebhookServiceName function here. 

```Golang
var WebhookServiceName = getWebhookServiceName()

// getWebhookServiceName - setting default WebhookServiceName
 func getWebhookServiceName() string {
 	webhookServiceName := os.Getenv("KYVERNO_SVC")
 	if webhookServiceName == "" {
 		webhookServiceName = "kyverno-svc"
 	}
 	return webhookServiceName
 }
```